lib/SSH/LibSSH.pm6

```diff
@@ -199,6 +200,7 @@ class SSH::LibSSH {
 
         submethod BUILD(Str :$!host!, Int :$!port = 22, Str :$!user = $*USER.Str,
                         Str :$!private-key-file = Str, Str :$!password = Str,
+                        Str :$!private-key-file-password = Str,
```

Comment:
The `= Str` is not needed here; it will already default to `Str`.

Reply:
Ah, and that was in the commit that went into `master`, which seems to have then been merged into this branch also, which is a bit messy.